procurement_purchase_no_grouping/__openerp__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 OdooMRP team
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Procurement Purchase No Grouping",
+    "version": "8.0.1.0.0",
+    "author": "OdooMRP team, AvanzOSC, Pedro M. Baeza, "
```

Comment:
Please change my name by Tecnativa, and you can remove `OdooMRP team`, as it's redundant.